components/basecomponents/StoryText.tsx

```diff
@@ -1,17 +1,19 @@
-import React, { useEffect, useRef, MutableRefObject } from 'react';
+import React, { MutableRefObject, useEffect, useRef } from 'react';
 import oPernikoveChaloupce from '../../data/translations/cs/pohadka_pernikovachaloupka.json';
 import oDvanactiMesickach from '../../data/translations/cs/pohadka_mesicky.json';
 import oCerveneKarkulce from '../../data/translations/cs/pohadka_karkulka.json';
 import oKoblizkovi from '../../data/translations/cs/pohadka_koblizek.json';
 import oIvasikovi from '../../data/translations/cs/pohadka_ivasik.json';
 import oHusach from '../../data/translations/cs/pohadka_husy.json';
 
+export type PhraseInfo = { language: string; time: number };
+
 interface StoryTextProps {
   audio: HTMLAudioElement | null;
   languageText: string;
```

Comment:
Consider renaming and retyping to  `textLanguage: Language; audioLanguage: Language`